camera/cameraSensorModule.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import os, random
+import robomodules as rm
+from messages import *
+import numpy as np
+import cv2
+import pickle
+
+ADDRESS = os.environ.get("BIND_ADDRESS","localhost")
+PORT = os.environ.get("BIND_PORT", 11297)
+
+FREQUENCY = 2
+
+class CameraSensorModule(rm.ProtoModule):
+    def __init__(self, addr, port, camera_port):
+        self.subscriptions = []
+        super().__init__(addr, port, message_buffers, MsgType, FREQUENCY)
+        self.camera_port = camera_port
+        self.cam = cv2.VideoCapture(self.camera_port)
+
+    def msg_received(self, msg, msg_type):
+        # This gets called whenever any message is received
+        # This module only sends data, so we ignore incoming messages
+        return
+
+    def tick(self):
+        # this function will get called in a loop with FREQUENCY frequency
+        # gets the camera feed, puts it into the message
+        msg = CameraFrameMsg()
+        ret, frame = self.cam.read()
+        # note: cameraFrame is apparently a string data type in the protocol buffer
+        msg.cameraFrame = pickle.dumps(frame)
+        msg = msg.SerializeToString()
+        self.write(msg, MsgType.CAMERA_FRAME_MSG)
+
+
+def main():
+    pass
```

Comment:
In order to actually run the module you need to make an instance of it here and run it:
```
module = CameraSensorModule(ADDRESS, PORT, CAM_PORT) #Previously set CAM_PORT to something
module.run()
```